Projects/Features/Scene/GroupDetailScene/GroupDetail/GroupDetailView.swift

```diff
@@ -0,0 +1,135 @@
+        NavigationBar(
+          type: .titleWithBackButtonAndIcon(store.groupDetail.groupName, DesignSystem.Icons.group),
+          backButtonAction: {
+            store.send(.backButtonDidTap)
+          },
+          rightIconAction: {
+            store.send(.memberListButtonDidTap)
+          }
+        )
+        
+        EventContainerView(eventDetail: store.groupDetail.eventDetail) { state in
+          store.send(.eventContainerViewButtonDidTap(state))
+        }
+        
+        dividerView
+      }
+      .overlay(ViewGeometry())
+      .onPreferenceChange(ViewHeightKey.self) { height in
+        self.bottomSheetHeight = height
+      }
+    }
+    .scrollIndicators(.hidden)
+    .sheet(isPresented: $showSheet) {
+      EventGridView(events: store.groupDetail.eventItems)
+        .scrollIndicators(.hidden)
+        .presentationDetents([.height(bottomSheetHeight), .large])
+        .interactiveDismissDisabled()
+        .presentationDragIndicator(.hidden)
+        .presentationBackgroundInteraction(.enabled(upThrough: .large))
+     }
+  }
+  
+  var dividerView: some View {
+    Rectangle()
+      .foregroundStyle(DesignSystem.Colors.gray20)
+      .frame(height: 10)
+  }
+}
+
+struct ViewHeightKey: PreferenceKey {
+  static var defaultValue: CGFloat = .zero
+  static func reduce(
+    value: inout CGFloat,
+    nextValue: () -> CGFloat
+  ) {
+    value = nextValue()
+  }
+}
+
+struct ViewGeometry: View {
+  var body: some View {
+    GeometryReader { geometry in
+      Color.clear
+        .preference(
+          key: ViewHeightKey.self,
+          // TODO: UIScreen 사용하지 않는 방향으로 개선 예정
+          value: UIScreen.main.bounds.height - geometry.size.height - UIScreen.topSafeArea - UIScreen.bottomSafeArea
+        )
+    }
+  }
+}
+
+// TODO: UIScreen 사용하지 않는 방향으로 개선 예정
+extension UIScreen {
+    static var topSafeArea: CGFloat {
+        let keyWindow = UIApplication.shared.connectedScenes
+            .filter({$0.activationState == .foregroundActive})
+            .map({$0 as? UIWindowScene})
+            .compactMap({$0})
+            .first?.windows
+            .filter({$0.isKeyWindow}).first
+        
+        return (keyWindow?.safeAreaInsets.top) ?? 0
+    }
+  
+  static var bottomSafeArea: CGFloat {
+      let keyWindow = UIApplication.shared.connectedScenes
+          .filter({$0.activationState == .foregroundActive})
+          .map({$0 as? UIWindowScene})
+          .compactMap({$0})
+          .first?.windows
+          .filter({$0.isKeyWindow}).first
+      
+      return (keyWindow?.safeAreaInsets.bottom) ?? 0
+  }
+}
+
+// 이벤트 목록 빈 경우
+#Preview {
+  GroupDetailView(
+    store: Store(
+      initialState: .init(groupDetail: GroupDetail.emptyMock),
+      reducer: {
+        GroupDetailCore()
+      }
+    )
+  )
+}
+
+// 이벤트 목록 있는 경우
+#Preview {
+  GroupDetailView(
+    store: Store(
+      initialState: .init(groupDetail: GroupDetail.mock),
+      reducer: {
+        GroupDetailCore()
+      }
+    )
+  )
+}
```

Comment:
P2: 이전 PR에서 아래와 같이 컨벤션을 맞춰서 store init 시 컨벤션 맞춰주세용❤️
```suggestion
#Preview {
  GroupDetailView(
    store: Store(
      initialState: .init(groupDetail: GroupDetail.emptyMock),
      reducer: GroupDetailCore.init
    )
  )
}

// 이벤트 목록 있는 경우
#Preview {
  GroupDetailView(
    store: Store(
      initialState: .init(groupDetail: GroupDetail.mock),
      reducer: GroupDetailCore.init
    )
  )
}
```

Reply:
@serena0720 요고 아마 템플릿에 GroupDetailCore() 이런식으로 돼있을거예요. 혹시 그 부분 수정해주실 수 있나요? 😋 앞으로 만들 프리뷰들을 위해 ㅎㅎㅎㅎ 현재 만들어져있는 부분들은 수정하겠습니다!

Reply:
싫다면요⁉️

Reply:
그럼 저도 수정안해요.

Reply:
싫다면요⁉️